### PR TITLE
#11341: Replace tt_lib in models/experimental/bert

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
Link to Github Issue #11341 
Master Issue #11240

### Problem description
Replace tt_lib with ttnn in models/experimental/bert

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/10418316955)
- [x] Model regression CI testing passes (if applicable)
- [x] New/Existing tests provide coverage for changes

### CODE REVIEW

File: models/experimental/bert/tt/add_and_norm.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```
Comment (KalaivaniMCW) [inline models/experimental/bert/tt/add_and_norm.py:1]: no need to change this I think. was this an auto change ?

File: models/experimental/bert/fused_ops/add_and_norm.py [hunk 2]
```diff
@@ -1,21 +1,21 @@
-# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```
Comment (KalaivaniMCW) [inline models/experimental/bert/fused_ops/add_and_norm.py:1]: here

**Review by arakhmati — APPROVED** [review]

**Review by arakhmati — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

File: models/experimental/bert/tt/mha.py [hunk 3]
```diff
@@ -109,17 +108,17 @@ def mha_(activation, attention_mask):
             W,
         ) = qkt.get_legacy_shape()  # Need to reshape right now since multi-C not supported for broadcast yet
         new_shape = [N, 1, C * H, W]
-        ttl.tensor.reshape(qkt, *new_shape)
+        ttnn.reshape(qkt, *new_shape)
```
Comment (KalaivaniMCW) [inline models/experimental/bert/tt/mha.py:111]: use ttnn.experimental for safer side. there were issues with ttnn reshape

File: models/experimental/bert/tt/mha.py [hunk 4]
```diff
@@ -109,17 +108,17 @@ def mha_(activation, attention_mask):
...
                 attention_score_input,
                 attention_mask,
-                ttl.tensor.BcastOpMath.ADD,
-                ttl.tensor.BcastOpDim.H,
+                ttnn.experimental.tensor.BcastOpMath.ADD,
+                ttnn.experimental.tensor.BcastOpDim.H,
             )
         attention_scores = softmax(attention_score_input)
-        ttl.tensor.reshape(attention_scores, N, C, H, W)  # Reshape back to original shape
+        ttnn.reshape(attention_scores, N, C, H, W)  # Reshape back to original shape
```
Comment (KalaivaniMCW) [inline models/experimental/bert/tt/mha.py:121]: use ttnn.experimental

**Review by KalaivaniMCW — APPROVED** [review]
Approved. Pls use ttnn.experimental.tensor.reshape wherever you have changed.

**Pull request merged**